package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "devDependencies": {
+    "lerna": "2.0.0-beta.32"
```

Comment:
Do we need to depends on the beta ? This is very boring to update.
> The instructions below are for Lerna 2.x which is currently in beta. We recommend using it instead of 1.x for a new Lerna project